Firestore/Example/Tests/Integration/API/FIRQueryTests.mm

```diff
@@ -29,6 +30,25 @@ @interface FIRQueryTests : FSTIntegrationTestCase
 
 @implementation FIRQueryTests
 
+/**
+ * Checks that running the query while online (against the backend/emulator) results in the same
+ * documents as running the query while offline. It also checks that both online and offline
+ * query result is equal to the expected documents.
+ *
+ * @param query The query to check.
+ * @param expectedDocs Array of document keys that are expected to match the query.
+ */
+- (void)checkOnlineAndOfflineQuery:(FIRQuery *)query matchesResult:(NSArray *)expectedDocs {
```

Comment:
In Objective C, optional parameter can only be implement by adding one more function. Since we always compare to `expectedDocs` in current test cases, I made `expectedDocs` required.

Reply:
👍